payments-be/src/test/java/pl/larp/larpex/pfsi/payment/adapter/mock/MockedConstants.java

```diff
@@ -0,0 +1,17 @@
+package pl.larp.larpex.pfsi.payment.adapter.mock;
+
+import java.util.UUID;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+
+// For now these are constants since we need to somehow work out how we should get this data
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public class MockedConstants {
+
+  public static final UUID EXISTING_USER_ID = UUID.fromString(
+    "8ebdb302-2589-4255-b060-d70d1bc974b8"
+  );
+  public static final UUID SOME_EVENT_ID = UUID.fromString(
```

Comment:
i get why userId is mocked, but why eventId?

Reply:
I mean it's a test so I mocked it

Reply:
I get it, the comment is misleading because it's not relevant anymore. I will remove it in a sec.

Reply:
right, my bad